misc/write_fake_manifests.py

```diff
@@ -6,7 +6,7 @@
 ones used in the Chromium project.
 
 Usage:
-  python misc/write_fake_manifests.py outdir  # Will run for about 5s.
+  ./misc/write_fake_manifests.py outdir  # Will run for about 5s.
```

Comment:
This is documentation, so it doesn't matter if the path to python isn't exactly correct. But I think it's probably technically wrong to suggest running the python script directly since that won't work on Windows, it's not a Windows executable.

Reply:
Windows does handle program names without file extensions by checking against a list of known extensions such as .exe, so running `python3` will detect a python3.exe and use that.

Of course, that only works for things Windows natively supports... exe files, batch scripts, vbs files, and funnily enough, JavaScript (but I believe JavaScript files will open in the windows scripting host, not something like Node.js

No generic scripting support, and that's despite the fact that running python3 on a stock install will open the windows store and prompt you to install python.

Reply:
ah ok great, sounds like this should be ok then

Reply:
The default name for Python 3 executable installed using official installer and not Windows Store is python.exe and not python3.exe

Reply:
it turns out cmake has a helper to discover python3, so I swapped to that which should solve this as well. Users can still explicitly set the variable if they'd prefer

Reply:
This particular review comment was about documentation usage text, anyway.

Reply:
oh sorry yea, but i think it applied to the main change too